docs/abi-spec.rst

```diff
@@ -141,19 +141,19 @@ on the type of ``X`` being
   ``enc(X) = head(X(1)) ... head(X(k)) tail(X(1)) ... tail(X(k))``
 
   where ``X = (X(1), ..., X(k))`` and
-  ``head`` and ``tail`` are defined for ``Ti`` being a static type as
+  ``head`` and ``tail`` are defined for ``Ti`` as follows:
+
+  if ``Ti`` is **static**:
```

Comment:
Why italics (or is it even bold)?

Reply:
It's **bold**. I felt like that word is the most important part in the sentence and deserved an emphases for fast and easy recognition

Reply:
Nothing else is bold anywhere and the sentence is short enough such that you immediately notice that `Ti` and `static` are the two most important parts in this sentence.

Reply:
> Nothing else is bold anywhere 

That's not necessarily a good thing :P Maybe more boldness would make it easier to read everywhere.

I dunno, I felt it reads nicer that way. But if you're against it, I'll remove it again.

Reply:
I think boldness distracts. Only headings should be bold. If you want to emphasize something, make it in italics, so you notice while you read it but not while you are looking at something else.

Reply:
I'm fine with it either way :-).